From: Diego Escalante Urrelo <diegoe@gnome.org>
Date: Thu, 23 Jul 2020 15:51:06 -0500
Subject: wl: Fix fall through warnings
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

Add a gcc special comment to solve a -Wimplicit-fallthrough warning:

src/wl/sys/wl_cfg80211_hybrid.h:52:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
   52 |  if (wl_dbg_level & WL_DBG_DBG) {   \
      |     ^
src/wl/sys/wl_cfg80211_hybrid.c:817:3: note: in expansion of macro ‘WL_DBG’
  817 |   WL_DBG(("network eap\n"));
      |   ^~~~~~
src/wl/sys/wl_cfg80211_hybrid.c:820:2: note: here
  820 |  default:
      |  ^~~~~~~
---
 amd64/src/wl/sys/wl_cfg80211_hybrid.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/amd64/src/wl/sys/wl_cfg80211_hybrid.c b/amd64/src/wl/sys/wl_cfg80211_hybrid.c
index c941bcc..03fba6e 100644
--- a/amd64/src/wl/sys/wl_cfg80211_hybrid.c
+++ b/amd64/src/wl/sys/wl_cfg80211_hybrid.c
@@ -795,6 +795,7 @@ wl_set_auth_type(struct net_device *dev, struct cfg80211_connect_params *sme)
 		break;
 	case NL80211_AUTHTYPE_NETWORK_EAP:
 		WL_DBG(("network eap\n"));
+        /* falls through */
 	default:
 		val = 2;
 		WL_ERR(("invalid auth type (%d)\n", sme->auth_type));
